lookup_tables/ipinfo/ipinfo_asn.yml

```diff
@@ -0,0 +1,292 @@
+AnalysisType: lookup_table
+LookupName: ipinfo_asn
+Schema: IPInfo.ASNCIDR
+Refresh:
+  RoleARN: arn:aws:iam::893421435052:role/panther-ipinfo-access-role
+  ObjectPath: s3://panther-ipinfo-ha771dhqm13j11czzb6kcpuy6ueccusw2a-s3alias/luts/data/ipinfo/asn.mmdb
+  PeriodMinutes: 60
+Description: IpInfo Owner (ASN) includes ASN details for every IP address and every ASNs related domains, allocation date, registry name, total number of IP addresses, and assigned prefixes. This dataset is for detection engine. For the datalake, also enable ipinfo_asn_datalake.
+Reference: https://docs.panther.com/enrichment/ipinfo
+Enabled: false
+LogTypeMap:
+  PrimaryKey: cidrblock
+  AssociatedLogTypes:
+    - LogType: AlphaSOC.Alert
+      Selectors:
+        - '$.event.srcIP'
+    - LogType: Amazon.EKS.Audit
+      Selectors:
```

Comment:
If I understand how all of this glues together, this is going to be running enrichment on an IP address that is a part of the EKS cluster and not the client/requestor's IP address. I think that it may be intended to check `$.sourceIPs[0]`

Reply:
or perhaps `.$sourceIPs`.. I see `p_any_ip_addresses` below which indicates to me that arrays are ok.

Reply:
I'm with you on that. List handling is a little wonky, but I think it will be fine for this case.

Reply:
cc @rleighton are we avoiding enriching lists or will it be fine here?

Reply:
Lists work

Reply:
ok, updated to `$.sourceIPs`